portafly/src/i18n/locales/en/shared.yml

```
@@ -128,3 +128,19 @@ loading:
   title_description: Title displayed as part of the loading state
   aria_label: Loading spinner  #-----TODO: revise this aria label
   aria_label_description: Aria label for the Loading component
+
+data_list_filter:
+  text_input:
+    aria_label: Filter text input
+    aria_label_description: Aria label for the filter text input
+    placeholder: "Filter by {{category}}"
+    placeholder_description: Filter by text input. This applies for filters by group/organization and admin
+  categories:
```

```suggestion
  options:
```

We're trying to consolidate to this term.